java/flight/flight-jdbc-driver/src/test/java/org/apache/arrow/driver/jdbc/ArrowFlightPreparedStatementTest.java

```diff
@@ -51,6 +52,7 @@ public static void setup() throws SQLException {
   @AfterClass
   public static void tearDown() throws SQLException {
     connection.close();
+    AutoCloseables.closeNoChecked(FLIGHT_SERVER_TEST_RULE);
```

Comment:
Doesn't this happen automatically because it's a ClassRule?

Reply:
You're right. Removed it.